webui/apps/eidolon-ui2/package.json

```diff
@@ -24,6 +24,12 @@
     "dev": "(patch -N ../../node_modules/.pnpm/@jsdevtools+ono@7.1.3/node_modules/@jsdevtools/ono/esm/index.js ono_turbo.patch || true) && npm run build-eidolon && next dev --turbo",
     "build": "npm run build-eidolon && next build",
     "build:static": "npm run build-eidolon && next build && next export",
+    "docker:up": "docker-compose up -d",
+    "docker:down": "docker-compose down",
+    "docker:build": "docker-compose build",
```

Comment:
this should probably build the dockerfile in sdk as well.

Reply:
Added/modified the following scripts:
```
    "docker:build-sdk": "docker build -t eidolonai/sdk_base:latest -f ../../../sdk/Dockerfile ../../../sdk/",
    "docker:build": "npm run docker:build-sdk && docker-compose build",
```